ignite/contrib/engines/common.py

```diff
@@ -320,15 +316,59 @@ def setup_mlflow_logging(trainer, optimizers=None, evaluators=None, log_every_it
     Returns:
         MLflowLogger
     """
-    mlflow_logger = MLflowLogger()
-    setup_any_logging(
-        mlflow_logger, mlflow_logger_module, trainer, optimizers, evaluators, log_every_iters=log_every_iters
-    )
-    return mlflow_logger
+    logger = MLflowLogger()
+    _setup_logging(logger, trainer, optimizers, evaluators, log_every_iters)
+    return logger
+
+
+def setup_neptune_logging(trainer, optimizers=None, evaluators=None, log_every_iters=100):
+    """Method to setup Neptune logging on trainer and a list of evaluators. Logged metrics are:
+        - Training metrics, e.g. running average loss values
+        - Learning rate(s)
+        - Evaluation metrics
+
+    Args:
+        trainer (Engine): trainer engine
+        optimizers (torch.optim.Optimizer or dict of torch.optim.Optimizer, optional): single or dictionary of
+            torch optimizers. If a dictionary, keys are used as tags arguments for logging.
+        evaluators (Engine or dict of Engine, optional): single or dictionary of evaluators. If a dictionary,
+            keys are used as tags arguments for logging.
+        log_every_iters (int, optional): interval for loggers attached to iteration events. To log every iteration,
+            value can be set to 1 or None.
+
+    Returns:
+        NeptuneLogger
+    """
+    logger = NeptuneLogger()
```

Comment:
I was thinking about those methods and loggers that need **kwargs to initialize. 
Either we need to pass those things or directly work on the instance of such logger...

Reply:
I was thinking about this, too. I think the tricky thing is that the `setup_X_logger` by convention passes in default kwargs. Adding `*logger_constructor_args` and `**logger_constructor_kwargs` isn't easy because of that. I think there are some clever things we could possibly do... will try to draft a gist quickly.

Reply:
See [here](https://gist.github.com/erip/42cb83e9a9b82e294cb338686f6563f6)

Reply:
Like here ?

https://github.com/pytorch/ignite/blob/662433798683ef72920f7ba4b248b16558df6adc/ignite/contrib/engines/common.py#L296

Reply:
I think that only works if it's a **kwargs-only constructor.

Reply:
> I think that only works if it's a **kwargs-only constructor.

Can’t test, I just have my phone 😅

Reply:
```python
>>> class Foo:
...     def __init__(a, b, c=None):
...         pass
...
>>> Foo(**dict(a=1, b=2, c=3))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() got multiple values for argument 'a'
```

Reply:
Ah, I was wrong. As long as we can guarantee the user specifies all constructor args they can pass it in as a dict at the end like in `VisdomLogger` even for positional args.

Reply:
So let’s define defaults and that will be ok 👌🏻

Reply:
Have #1015 out :-)